config.nims

```diff
@@ -0,0 +1,4 @@
+# begin Nimble config (version 1)
+when fileExists("nimble.paths"):
+  include "nimble.paths"
```

Comment:
@zah we need a solution the random paths being used, before we start spamming all the repos with these broken files